spec/spec_helper.rb

```diff
@@ -17,3 +27,33 @@
   config.order = "random"
   config.infer_spec_type_from_file_location!
 end
+
+Geocoder.configure(:lookup => :test)
+
+Geocoder::Lookup::Test.add_stub(
+  "Statue of Liberty, NY, USA", [
+    {
+      'latitude'     => 40.6892494,
+      'longitude'    => -74.0445003,
+      'address'      => "Statue of Liberty National Monument, New York, NY, USA",
+      'state'        => 'New York',
+      'state_code'   => 'NY',
+      'country'      => 'United States',
+      'country_code' => 'US'
+    }
+  ]
+)
+
+Geocoder::Lookup::Test.set_default_stub(
+  [
+    {
+      'latitude'     => 40.7143528,
+      'longitude'    => -74.0059731,
+      'address'      => 'New York, NY, USA',
+      'state'        => 'New York',
+      'state_code'   => 'NY',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.